docs/conf.py

```diff
@@ -50,6 +50,7 @@
     'sphinx.ext.napoleon',
     'sphinx.ext.autodoc',
     'sphinx_tabs.tabs',
+    'sphinx.ext.autosectionlabel',
```

Comment:
Please add this new dependency into the `conda/unix.yml` file

Reply:
this is a builtin extension like sphinx.ext.autodoc so I believe this is not required.

Reply:
OK